Sources/SQLTokenizer.swift

```diff
@@ -0,0 +1,270 @@
+import Foundation
+
+public class SQLTokenizer {
+    internal enum Token: String {
+        case Error = "Error"
+        case Hexadecimal = "Hexadecimal"
+        case Identifier = "Identifer"
+        case Integer = "Integer"
+        case Keyword = "Keyword"
+        case Literal = "Literal"
+        case Operator = "Operator"
+        case Parameter = "Parameter"
+        case Real = "Real"
+        case Terminal = "Terminal"
+        case Whitespace = "Whitespace"
+    }
+    
+    private(set) var items: [[String: String]] = []
```

Comment:
I feel like this would be more descriptive if it was something like:
```
struct TokenValue {
    let token: Token
    let value: String
}

private(set) var items: [TokenValue] = []
```

Then you don't need to store the rawValue of the token in a dictionary to satisfy the type system, and you wouldn't need to unwrap the values in the dictionary.